frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -20,24 +20,41 @@ const register = (): WorkflowConfiguration => {
         description: "A searchable catalog of services",
         component: Catalog,
         featureFlag: "projectCatalog",
+        layoutProps: {
+          variant: "standard",
+          title: "Project Catalog",
```

Comment:
For the title and the subtitle, I agree that it is redundant with the other config props. I initially defined it as so, because  some of the workflows define their own in a component, ignoring the `heading` prop. 

We could definitely just use the other attributes and leaving the `breadcrumbsOnly` prop, because the title and subtitle will be always defined (from what I've seen in the workflows config).